Core1Platform/UserInterface/01d_U2F.md

```diff
@@ -0,0 +1,55 @@
+[!!Set up U2F from the user side](../UsingCore1/01b_U2FLogin.md)   
+[!!Set up U2F from the administrator side](../AdministratingCore1/03_AdminSetupActindo.md)
+
```

Comment:
Gibt es Leerzeichen zwischen den beiden Links? Die werden in zwei Zeilen angezeigt. Bitte prüfen und ggf. Leerzeichen/Trennung löschen, sonst werden sie auf der Webseite nicht richtig angezeigt.
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/9134c4d2-90e0-4bf5-a79e-9d00d2f3aa17)

Reply:
done